en/sensor/sfxx_lidar.md

```diff
@@ -0,0 +1,43 @@
+# LightWare SF1XX Lidar
+
+LightWare develops a range of light-weight, general purpose, laser altimeters ("Lidar") suitable for mounting on UAVs. These are useful for applications including terrain following, precision hovering (e.g. for photography), warning of regulatory height limits, anti-collision sensing etc.
+
+![LightWare SF11/C Lidar](../../assets/hardware/sensors/sf11c_120_m.jpg)
+
+PX4 supports the following LightWare Lidar rangefinders through the I2C bus (only):
+* [SF10/A](http://lightware.co.za/shop2017/drone-altimeters/26-sf10a-25-m.html) (25 m)
+* [SF10/B](http://lightware.co.za/shop2017/drone-altimeters/25-sf10b-50-m.html) (50 m)
+* SF10/C (100m) (Discontinued)
```

Comment:
FYI, I can't find any mention of this on their site , so I believe it is discontinued.

Reply:
Leaving marked as discontinued. It certainly isn't available.